internal/controllers/inboundOrders/create.go

```diff
@@ -0,0 +1,91 @@
+package inboundOrders_controller
```

Comment:
decidimos usar snake_case como padrão pra pastas e packages

```suggestion
package inbound_orders_controller
```